src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String BUY_CONSTANT = "buy";
+    private static final String COMMA = ",";
+    private static final String RESULT_CONSTANT = "result";
+    private static final String SUPPLY_CONSTANT = "supply";
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
+    public void getStatistic(String fromFileName, String newFile) {
+        int[] stats = readStatistic(fromFileName);
+        String result = countStatistic(fromFileName, stats);
+        writeStatistic(result, newFile);
+        System.out.println(result);
```

Comment:
```suggestion
```

Reply:
I need to do report. I guessed it should be done in console. I have no need to do it or I should do it in main method?